### DISCUSSION TITLE
Release v10.0.0

### DISCUSSION THREAD

**Discussion by jessesquires** [Quick/Nimble#979] [Announcements]
Hey folks, some new maintainers (including myself) have been added to this project. We are working on getting the v10 release out as soon as we can. Your help with pull requests and reviews is appreciated.

Regarding the status of this project: yes this project has suffered from neglect for awhile. Many original maintainers have moved on. 

Remember, we are all volunteering for free here, so please don't be an asshole! 😎 ✌🏼 

Tracking Milestone:
https://github.com/Quick/Nimble/milestone/11

**Comment by jessesquires** [comment]
Hey @younata! 👋🏼 

Correct me if I'm wrong, but it looks like you are eager to help out here and get this release finished. 😄 

- I checked the diff from the last release: https://github.com/Quick/Nimble/compare/v9.2.1...main
- I added the v10 milestone to all changes: https://github.com/Quick/Nimble/milestone/11
- I also did a bunch of clean up and made all the various PR status checks required, etc.

If you are up for it, let's coordinate here! 😊 

I think the only thing to call out is making sure we add the v10 milestone to everything. It would also be nice to label PRs that fix bugs, add features, or introduce breaking changes, so that we can write the release notes.

**Reply by younata** [reply]
Hey @jessesquires!

I am 100% down to help with this. I'll get to adding labels and labeling PRs as necessary.

**Comment by younata** [comment]
I'm going to move watchOS support to v11. That can also coincide with the next major release of Quick.

**Comment by jessesquires** [comment]
Thanks @younata ! 💯 ❤️

**Comment by jessesquires** [comment]
v10 released:
https://github.com/Quick/Nimble/releases/tag/v10.0.0
